pkgs/servers/gotosocial/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "gotosocial";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "superseriousbusiness";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-85UmO2ncPMSVbwmeYFWnZ39qZSfsQ+pFovmzhzdnQYo=";
+  };
+
+  vendorHash = null;
+
+  meta = with lib; {
+    homepage = "https://docs.gotosocial.org/";
+    description = "Fast, fun, ActivityPub server, powered by Go";
+    longDescription = ''
+      ActivityPub social network server, written in Golang.
+      You can keep in touch with your friends, post, read, and
+      share images and articles. All without being tracked or
+      advertised to! A light-weight alternative to Mastodon
+      and Pleroma, with support for clients!
+    '';
+    maintainers = with maintainers; [ pbsds ];
```

Comment:
@pbsds as a suggestion, based on your comment [here](https://github.com/NixOS/nixpkgs/pull/202110#issuecomment-1442428541) you may want to take @fleaz offer [here](https://github.com/NixOS/nixpkgs/pull/202110#issuecomment-1442033253)

Reply:
Although not spelled out explicitly, i did give the go ahead. I uploaded my newest version with this in mind

Reply:
Sorry for the delay, i fell off a ladder and broke my arm 🙈  

Will start to work on this soon.

Reply:
Best wishes!